<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mnemo</class>
 <widget class="QLabel" name="mnemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1304</width>
    <height>708</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1304</width>
    <height>708</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>mnemo</string>
  </property>
  <property name="pixmap">
   <pixmap resource="mnemo/mnemo.qrc">:/mnemo/mnemo.svg</pixmap>
  </property>
  <property name="alignment">
   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
  </property>
  <widget class="QStackedWidget" name="kf_1">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>75</y>
     <width>300</width>
     <height>306</height>
    </rect>
   </property>
  </widget>
  <widget class="QStackedWidget" name="kf_2">
   <property name="geometry">
    <rect>
     <x>465</x>
     <y>75</y>
     <width>300</width>
     <height>306</height>
    </rect>
   </property>
  </widget>
  <widget class="QStackedWidget" name="kf_3">
   <property name="geometry">
    <rect>
     <x>829</x>
     <y>75</y>
     <width>300</width>
     <height>306</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_Pair">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Тиск технологічного повітря</string>
   </property>
   <property name="text">
    <string>999</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_Pvoda">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>30</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Тиск води</string>
   </property>
   <property name="text">
    <string>999</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_Gsusp">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>480</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Витрата суспензії</string>
   </property>
   <property name="text">
    <string>999</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_Gvoda">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>420</y>
     <width>40</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Витрата води</string>
   </property>
   <property name="text">
    <string>999</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_Lsusp">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>590</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Рівень в зб суспензії</string>
   </property>
   <property name="text">
    <string>999</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_Lp">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>595</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Рівень в зб. промою</string>
   </property>
   <property name="text">
    <string>999</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_Lcs">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>596</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Рівень в зб. чистого соку</string>
   </property>
   <property name="text">
    <string>999</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_Lvoda">
   <property name="geometry">
    <rect>
     <x>1183</x>
     <y>598</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Рівень в зб. води</string>
   </property>
   <property name="text">
    <string>999</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="pb_Lsusp">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>620</y>
     <width>17</width>
     <height>67</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>212</red>
         <green>211</green>
         <blue>211</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>212</red>
         <green>211</green>
         <blue>211</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>212</red>
         <green>211</green>
         <blue>211</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>212</red>
         <green>211</green>
         <blue>211</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="toolTip">
    <string>Рівень в зб. суспензії</string>
   </property>
   <property name="maximum">
    <number>4000</number>
   </property>
   <property name="value">
    <number>400</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QProgressBar" name="pb_Lp">
   <property name="geometry">
    <rect>
     <x>204</x>
     <y>620</y>
     <width>17</width>
     <height>67</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>212</red>
         <green>211</green>
         <blue>211</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>212</red>
         <green>211</green>
         <blue>211</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>212</red>
         <green>211</green>
         <blue>211</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>212</red>
         <green>211</green>
         <blue>211</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="toolTip">
    <string>Рівень в зб. промою</string>
   </property>
   <property name="maximum">
    <number>4000</number>
   </property>
   <property name="value">
    <number>400</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QProgressBar" name="pb_Lcs">
   <property name="geometry">
    <rect>
     <x>1105</x>
     <y>620</y>
     <width>17</width>
     <height>67</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>212</red>
         <green>211</green>
         <blue>211</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>212</red>
         <green>211</green>
         <blue>211</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>212</red>
         <green>211</green>
         <blue>211</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>212</red>
         <green>211</green>
         <blue>211</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="toolTip">
    <string>Рівень в зб. чистого соку</string>
   </property>
   <property name="maximum">
    <number>4000</number>
   </property>
   <property name="value">
    <number>400</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QProgressBar" name="pb_Lvoda">
   <property name="geometry">
    <rect>
     <x>1236</x>
     <y>620</y>
     <width>17</width>
     <height>67</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>212</red>
         <green>211</green>
         <blue>211</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>212</red>
         <green>211</green>
         <blue>211</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>212</red>
         <green>211</green>
         <blue>211</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>212</red>
         <green>211</green>
         <blue>211</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="toolTip">
    <string>Рівень в зб води</string>
   </property>
   <property name="maximum">
    <number>4000</number>
   </property>
   <property name="value">
    <number>400</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_Tvoda">
   <property name="geometry">
    <rect>
     <x>1266</x>
     <y>620</y>
     <width>40</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Температура води</string>
   </property>
   <property name="text">
    <string>999</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_Nps">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>670</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Потужніст насосу суспензії</string>
   </property>
   <property name="text">
    <string>999</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_Npv">
   <property name="geometry">
    <rect>
     <x>1154</x>
     <y>670</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Потужність насосу води</string>
   </property>
   <property name="text">
    <string>999</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_Xv_01">
   <property name="geometry">
    <rect>
     <x>93</x>
     <y>630</y>
     <width>40</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_Xv_02">
   <property name="geometry">
    <rect>
     <x>1183</x>
     <y>635</y>
     <width>40</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_Y_08">
   <property name="geometry">
    <rect>
     <x>1198</x>
     <y>554</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
min-width: 16px;
min-height: 16px;
max-width: 16px;
max-height: 16px;
}
QCheckBox:checked
{
	image: url(:/valves/lamp_green_16x16.png);
}

QCheckBox:unchecked
{
	image: url(:/valves/lamp_off_16x16.png);
}

QCheckBox:indicator
{
	background: rgba(0, 0, 0, 0);
	height: 16px;
	width: 16px;
}
</string>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_Y_09">
   <property name="geometry">
    <rect>
     <x>1271</x>
     <y>555</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
min-width: 16px;
min-height: 16px;
max-width: 16px;
max-height: 16px;
}
QCheckBox:checked
{
	image: url(:/valves/lamp_green_16x16.png);
}

QCheckBox:unchecked
{
	image: url(:/valves/lamp_off_16x16.png);
}

QCheckBox:indicator
{
	background: rgba(0, 0, 0, 0);
	height: 16px;
	width: 16px;
}
</string>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
  <widget class="QFrame" name="frameTr">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>430</y>
     <width>711</width>
     <height>271</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_State_1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Зупинено</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_State_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>20</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Зупинено</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_State_3">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>10</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Зупинено</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_AM_01">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>640</y>
     <width>18</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
min-width: 18px;
min-height: 20px;
max-width: 18px;
max-height: 20px;
}
QCheckBox:unchecked
{
	image: url(:/butons/pict/hand_18x20.png);
}
QCheckBox:indicator
{
	background: rgba(0, 0, 0, 0);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cX_01">
   <property name="geometry">
    <rect>
     <x>103</x>
     <y>657</y>
     <width>26</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
min-width: 26px;
min-height: 30px;
max-width: 26px;
max-height: 30px;
}

QPushButton
{
	border-image: url(:/butons/pict/lib/pump_off_26x30.png);
}
QPushButton:pressed
{
	border-image: url(:/butons/pict/lib/pump_on_26x30.png);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_AM_02">
   <property name="geometry">
    <rect>
     <x>1168</x>
     <y>640</y>
     <width>18</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
min-width: 18px;
min-height: 20px;
max-width: 18px;
max-height: 20px;
}
QCheckBox:unchecked
{
	image: url(:/butons/pict/hand_18x20.png);
}
QCheckBox:indicator
{
	background: rgba(0, 0, 0, 0);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cX_02">
   <property name="geometry">
    <rect>
     <x>1189</x>
     <y>659</y>
     <width>26</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
min-width: 26px;
min-height: 30px;
max-width: 26px;
max-height: 30px;
}

QPushButton
{
	border-image: url(:/butons/pict/lib/pump_off_l_26x30.png);
}
QPushButton:pressed
{
	border-image: url(:/butons/pict/lib/pump_on_l_26x30.png);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_AM_Tv">
   <property name="geometry">
    <rect>
     <x>1234</x>
     <y>520</y>
     <width>18</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{
min-width: 18px;
min-height: 20px;
max-width: 18px;
max-height: 20px;
}
QCheckBox:unchecked
{
	image: url(:/butons/pict/hand_18x20.png);
}
QCheckBox:indicator
{
	background: rgba(0, 0, 0, 0);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="bnY_08">
   <property name="geometry">
    <rect>
     <x>1213</x>
     <y>541</y>
     <width>27</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../../pict/lib/valves.qrc">
     <normaloff>:/valves/valve_off_l_20x32.png</normaloff>:/valves/valve_off_l_20x32.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="bnY_09">
   <property name="geometry">
    <rect>
     <x>1246</x>
     <y>542</y>
     <width>27</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../../pict/lib/valves.qrc">
     <normaloff>:/valves/valve_off_20x32.png</normaloff>:/valves/valve_off_20x32.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>32</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="mnemo/mnemo.qrc"/>
  <include location="../../../lib/picture.qrc"/>
  <include location="../../../pict/lib/valves.qrc"/>
 </resources>
 <connections/>
</ui>
